### DISCUSSION TITLE
Reduce logging footprint -- reduce significant figures by rounding floats in logs

### DISCUSSION THREAD

**Discussion by jkrauska** [MinaProtocol/mina#9586] [Ideas]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/MinaProtocol/mina/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/MinaProtocol/mina/discussions

### Description

Every few seconds I see log lines that could be a lot simplier.

They are using up lots of bytes in logs with irrelevant data -- round some floats please.

eg:
2021-10-03 19:50:03 UTC [Info] Updating new available work took 5.097389221191406 ms

should really be written at most as
2021-10-03 19:50:03 UTC [Info] Updating new available work took 5.097 ms

### Steps to Reproduce

1. Run a node
2. Watch logs
3. Think about what's relevant

### Expected Result

2021-10-03 19:50:03 UTC [Info] Updating new available work took 5.097 ms

### Actual Result

2021-10-03 19:50:03 UTC [Info] Updating new available work took 5.097389221191406 ms

### How frequently do you see this issue?

Always

### What is the impact of this issue on your ability to run a node?

Low

### Status

```shell
NA
```

### Additional information

code:
https://github.com/MinaProtocol/mina/blob/b137fbd750d9de1b5dfe009c12de134de0eb7200/src/lib/work_selector/work_lib.ml#L79

rounding floats:
https://discuss.ocaml.org/t/rounding-floats-to-number-of-decimals/6921/3

**Comment by joseandro** [comment]
Thanks for your suggestion @jkrauska. I will convert this Issue into a Discussion so that it can be properly discussed between the community.
